ibis/expr/schema.py

```diff
@@ -153,6 +153,34 @@ def from_tuples(
         # construct the schema
         return cls(dict(zip(names, types)))
 
+    @classmethod
+    def from_numpy(self, numpy_schema):
+        """Return the equivalent ibis schema."""
+        from ibis.formats.numpy import schema_from_numpy
+
+        return schema_from_numpy(numpy_schema)
+
+    @classmethod
+    def from_pandas(self, pandas_schema):
+        """Return the equivalent ibis schema."""
+        from ibis.formats.pandas import schema_from_pandas
+
+        return schema_from_pandas(pandas_schema)
+
+    @classmethod
+    def from_pyarrow(self, pyarrow_schema):
+        """Return the equivalent ibis schema."""
+        from ibis.formats.pyarrow import schema_from_pyarrow
+
+        return schema_from_pyarrow(pyarrow_schema)
+
+    @classmethod
+    def from_dask(self, dask_schema):
+        """Return the equivalent ibis schema."""
+        from ibis.formats.dask import schema_from_dask
+
+        return schema_from_dask(dask_schema)
```

Comment:
Looks like maybe this module is missing some test coverage.

Reply:
I'm removing the whole `schema_from_dask` in a follow-up :)

Reply:
Covered the method in a test.